tests/data_test.py

```diff
@@ -30,15 +32,48 @@ def test_datasets(ds_cls: type[VisionDataset]) -> None:
 
 
 def test_audio_dataset() -> None:
-    """Tests basic functionality of an audio dataset base class."""
+    """Tests basic functionality of the base audio dataset class."""
     x = torch.rand(1, 10)
     audio_dir = Path(r"Sample path")
     dataset = PBAudioDataset(x=x, audio_dir=audio_dir)
 
     assert dataset is not None
     assert len(dataset) == len(x)
+
+    
+def test_audio_dataset() -> None:
```

Comment:
can't be named the same as another test.

Reply:
Add `@pytest.mark.slow`

Reply:
I didn't even realise I did this wow lol, I might just remove the test for the base class? Doesn't do much?